src/JBrowse/Browser.js

```diff
@@ -320,6 +320,11 @@ initPlugins: function() {
             return new Deferred();
         });
 
+        require.on("error", function(error) {
+            console.error(error);
+            deferred.reject('A resource failed to load '+error.src + ':' + error.info[0]);
+        });
+
```

Comment:
Could you de-install this error handler after the plugin loading is finished?  It's specific to the plugin loading, and `require` is a persistent global thing.

If something else fails to load later, it will try again to reject the plugin-loading deferred.

Reply:
Found from this thread that it can return a handle that can be used to later remove it https://github.com/dojo/loader/issues/14

This allows the glyph error handler to be separate at the later stage (added that too)

Reply:
This is reverted now, I think just having a global require.on error handler in index.html that creates an error screen sort of a better approach. Potentially this is a downside because if one track fails to load it just jumps to the error screen rather than just continuing to display other tracks, but the idea of adding and removing the error handler didnt seem to be working.

Also the "not-a-module" error can be caught now